tests/src/test/java/alluxio/master/FileSystemMasterClientIntegrationTest.java

```diff
@@ -32,7 +35,8 @@
 public final class FileSystemMasterClientIntegrationTest {
   @Rule
   public LocalAlluxioClusterResource mLocalAlluxioClusterResource =
-      new LocalAlluxioClusterResource.Builder().build();
+      new LocalAlluxioClusterResource.Builder().setProperty(PropertyKey.MASTER_RETRY, 30)
+          .setProperty(PropertyKey.MASTER_RPC_PORT, 19977).build();
```

Comment:
I was over worried. This is not a problem as we reset the master port after we got the random port (DefaultAlluxioMaster.java, line 136).

Removed